core/db/migrate/20210512191732_create_spree_cms_pages.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSpreeCmsPages < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_cms_pages do |t|
+      t.column :title, :string, null: false
+      t.column :meta_title, :string
+      t.column :content, :text
+      t.column :meta_description, :text
+      t.column :visible, :boolean, default: false
```

Comment:
@MatthewKennedy let's add an index for this

Reply:
@damianlegawiec Not a problem, I'll be on with this over the weekend.